azure-pipelines.yml

```diff
@@ -142,8 +142,13 @@ jobs:
         - Windows_NT_x86
       ${{ if in(variables['Build.DefinitionName'], 'coreclr-corefx', 'coreclr-corefx-jitstress', 'coreclr-corefx-jitstressregs', 'coreclr-corefx-jitstress2-jitstressregs') }}:
         platforms:
+        # TODO: add Windows_NT_arm64, when we have hardware available, and .NET Core supports it. Note: platform-matrix.yml needs to enable a Helix queue for this.
+        - Linux_arm
+        - Linux_arm64
         - Linux_x64
```

Comment:
Are you planning to add macOS?

Reply:
Not currently. I don't think it would significantly increase our coverage, and we don't need to introduce any more potential infrastructure instability. We can change our mind as we gain experience with this testing.